packages/server/lib/modes/run.js

```diff
@@ -95,6 +95,15 @@ const formatFooterSummary = function (results) {
   ]
 }
 
+const formatNodeVersion = ({ resolvedNodeVersion, resolvedNodePath }) => {
+  debug('formatting Node version. Resolved version "%s", path if any "%s"',
```

Comment:
Use `%o` with key/value instead of `%s` so there's more context as to where the values come from.